scripts/index.js

```diff
@@ -1 +1,42 @@
-console.log(`There are ${characters.length} characters in the array.`);
\ No newline at end of file
+// console.log(`There are ${characters.length} characters in the array.`);
+
+console.log (characters[0].died)
+
+// How many character names start with A
+// function beginsWithA(character) {
+//     return character.name.startsWith('A');
+// }
+
+// function findsACharacters(book) {
+//     aCharacters = book.filter(beginsWithA)
+//     return aCharacters.length
+// }
+// console.log(findsACharacters(characters))
+
+
+// How many character names start with Z
+// function beginsWithZ(character) {
+//     return character.name.startsWith('Z');
+// }
+
+// function findsACharacters(book) {
+//     aCharacters = book.filter(beginsWithZ)
+//     return aCharacters.length
+// }
+// console.log(findsACharacters(characters))
+
+// //How many characters are dead?
+deadCharacters = characters.forEach(function(key) {
```

Comment:
Hey Eric were you able to get this function to output the expected number? I wasn't able to on my end. I tried to use most of the code you started with. This is a solution I tried to implement 

`function deadCount(characters) {
    let count = 0;
    characters.forEach(function(key) {
        if (key.died !== '') {
            count += 1;
        }
    })
    return count;
}

console.log(deadCount(characters))`

Let me know what you think. If yours was working on your end great job. I may have had a different set up.

Reply:
Hi April,

Yes, I am getting a return of 553 with both your solution and mine. See below for what I utilized. Thanks again for reviewing!

`function deadCharacters (characterList) {
    let deadArray = characterList.filter(function(personName) {
        return personName.died !== '';
    })
    return deadArray.length;
}
console.log(deadCharacters(characters));`